### PR TITLE
Support for UEFI Secure Boot 

### PR DESCRIPTION
**Pull request by ricardosalveti**
PR to validate against https://github.com/foundriesio/meta-lmp/pull/682.

### CODE REVIEW

**Review by MrCry0 — APPROVED** [review]
LGTM

File: conf/local.conf [hunk 1]
```diff
@@ -81,3 +81,10 @@ UBOOT_SIGN_KEYNAME ?= "ubootdev"
 # custom signing key just generate a custom RSA 2048 key (PEM format) and
 # set via the OPTEE_TA_SIGN_KEY variable.
 OPTEE_TA_SIGN_KEY ??= "${TOPDIR}/conf/keys/opteedev.key"
+
+#
+# UEFI Secure Boot
+#
+# Folder for UEFI keys and certificates
+UEFI_SIGN_KEYDIR ??= "${TOPDIR}/conf/keys/uefi"
+#UEFI_SIGN_ENABLE ?= "1"
```
Comment (quaresmajose) [inline conf/local.conf:90]: To validate the PR  https://github.com/foundriesio/meta-lmp/pull/682 we need this bit enabled or not?

**Review by angolini — APPROVED** [review]
Assuming the last commit (personal repo) is for testing only and will not be merged, LGTM

File: conf/local.conf (hunk 1 above)
Comment (ricardosalveti) [reply conf/local.conf:90]: It is enabled for intel only (at meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc):

```
# Intel
UEFI_SIGN_ENABLE:intel-corei7-64 = "1"
```

**Comment by ricardosalveti** [comment]
> Assuming the last commit (personal repo) is for testing only and will not be merged, LGTM

Yes, will push the change using the updated meta-lmp now.

File: conf/local.conf (hunk 1 above)
Comment (quaresmajose) [reply conf/local.conf:90]: right

**Review by Tim-Anderson — APPROVED** [review]
LGTM

**Pull request merged**